packages/stencil/components/input/src/components/osds-input/osds-input.scss

```diff
@@ -49,14 +58,6 @@ input[type=number]::-webkit-inner-spin-button {
   margin: 0;
 }
 
-[disabled] {
-  @include osds-input-on-selected-host() {
-    opacity: .5;
-    cursor: not-allowed;
-    pointer-events: none;
```

Comment:
is it what has been asked for? cause this now allows user to select content on disabled input which was not doable before

Reply:
goal was to display cursor as not allowed but not be able to interact with the input (which is enabled by the native disabled attribute of it)